xtevent/xtevent.sthlp

```diff
@@ -72,7 +74,7 @@ See {help xteventtest} for hypothesis testing after estimation and {help xtevent
 {pstd}
 {cmd: xtevent} estimates the effect of a policy variable of interest on a dependent variable using a panel event
 study design. Additional control variables can be included in {it:varlist}. The command allows for estimation when a pre-trend is present using
-the instrumental variables estimator of Freyaldenhoven et al. (2019). {p_end}
+the instrumental variables estimator of Freyaldenhoven et al. (2019). It also allows to use the Interaction Weighted Estimator of Sun and Abraham (2021).{p_end}

```

Comment:
@Constantino-Carreto-Romero "It also allows estimation in settings with heterogeneous effects by cohort using the Interaction Weighted ..."

Reply:
I implemented it in https://github.com/JMSLab/xtevent/commit/cb067f397837feaeeef0495750f70c3a25bf43a8

Reply:
@Constantino-Carreto-Romero Please fix the typo.

Reply:
I removed the extra space in https://github.com/JMSLab/xtevent/commit/22b3ea79edf6e8d30fa3c7e854cc00ce4c07da29